edge/pkg/edgehub/process.go

```diff
@@ -81,11 +81,20 @@ func (eh *EdgeHub) sendToKeepChannel(message model.Message) error {
 }
 
 func (eh *EdgeHub) dispatch(message model.Message) error {
-	// TODO: dispatch message by the message type
-	md, ok := groupMap[message.GetGroup()]
-	if !ok {
-		klog.Warningf("msg_group not found")
-		return fmt.Errorf("msg_group not found")
+	group := message.GetGroup()
+	md := ""
+	switch group {
+	case messagepkg.ResourceGroupName:
+		md = modules.MetaGroup
+	case messagepkg.TwinGroupName:
+		md = modules.TwinGroup
+	case messagepkg.FuncGroupName:
+		md = modules.MetaGroup
+	case messagepkg.UserGroupName:
+		md = modules.BusGroup
+	default:
+		klog.Warningf("msg_group: %s not found", group)
+		return fmt.Errorf("msg_group: %s not found", group)
```

Comment:
What's the difference of these two implementation?

Reply:
Maybe the groupMap can be deleted?

Reply:
Use switch-case looks more intuitive, the groupMap structure still  used in edgehub currently.